### ISSUE TITLE
Dockerizing a Node.js web app: inaccuracy in the guide

### ISSUE THREAD

**Issue by Alexhha** [nodejs/nodejs.org#1234]
* **URL**: https://nodejs.org/uk/docs/guides/nodejs-docker-webapp/
* **Browser version**: Google Chrome 56.0.2924.87
* **Operating system**: Ubuntu 16.04

There is an inaccuracy in the guide above. When you start server.js application Express framework will be listening on all interfaces (0.0.0.0) and not on localhost as you are printed to the console
```js
console.log('Running on http://localhost:' + PORT);
```

And that's why the application will be available outside of the docker container. It's a crucial detail to run and access nodejs application outside of the docker. 

It would be grate to state this moment in the example

```js
const PORT = 8080;
const HOST = '0.0.0.0';
...
app.listen(PORT, HOST);
console.log('Running on http://' + HOST + ':' + PORT);
```

**Comment by fhemberger** [comment]
Good point!

Could you please write up a a pull request for it?

**Comment by fhemberger** [comment]
Fixed via #1274.
